src/Deprecated11/SHRBTextStyler.extension.st

```diff
@@ -0,0 +1,23 @@
+Extension { #name : #SHRBTextStyler }
+
+{ #category : #'*Deprecated11' }
```

Comment:
Deprecated12?

Reply:
Yeah I originally did this Pr on P11 and I need to update it. I’ll do it today